api/generic-cti/newcall-event.rst

```diff
@@ -0,0 +1,269 @@
+New call
```

Comment:
Proposal for the headline: Event: newCall

Reply:
What do you think, Gerrit?